pkg/yamls/yamls.go

```diff
@@ -0,0 +1,159 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package embeddedyamls
+
+import _ "embed"
+
+//go:embed deploy/crds/submariner.io_brokers.yaml
```

Comment:
Embed all files into a file system.

Eg:

import "embed"
// content is our static web server content.
//go:embed image template html/index.html
var content embed.FS

Reply:
The calling functions expect a string. So embed them as a string as you are currently doing.

Reply:
Right, and this would also avoid the chicken-and-egg problem...